specs/flow_defs_reorganization.adoc

```diff
@@ -0,0 +1,287 @@
+          model:
+            type: String
+          model_name:
+            type: String
+          vendor_id:
+            type: String
+        enabled: true
+        value: nodes/$Node_context.node_id/Cpu
+      Memory:
+        attrs:
+          total_size:
+            type: String
+          total_swap:
+            type: String
+        enabled: true
+        value: nodes/$Node_context.node_id/Memory
+      Node:
+        atoms:
+          cmd:
+            enabled: true
+            inputs:
+              mandatory:
+                - Node.cmd_str
+            name: "Execute CMD on Node"
+            run: tendrl.node_agent.atoms.node.cmd.Cmd
+            type: Create
+            uuid: dc8fff3a-34d9-4786-9282-55eff6abb6c3
+          check_node_up:
+            enabled: true
+            inputs:
+              mandatory:
+                - Node.fqdn
+            outputs:
+              - Node.status
+            name: "check whether the node is up"
+            run: tendrl.node_agent.atoms.node.check_node_up
+            type: Create
+            uuid: eda0b13a-7362-48d5-b5ca-4b6d6533a5ab
+        attrs:
+          cmd_str:
+            type: String
+          fqdn:
+            type: String
+          status:
+            type: Boolean
+        enabled: true
+        value: nodes/$Node_context.node_id/Node
+      OS:
+        attrs:
+          kernel_version:
+            type: String
+          os:
+            type: String
+          os_varsion:
+            type: String
+          selinux_mode:
+            type: String
+        enabled: true
+        value: nodes/$Node_context.node_id/Os
+  namespace.tendrl.integration.ceph:
+    version_bindings:
+      ceph-x.a.b : ceph-integration-y.e.f
+      .
+      .
+      .
+    provisioning:
+      pip:
+        .
+        .
+        .
+  namespace.tendrl.integration.gluster:
+    version_bindings:
+      gluster-x.a.b : gluster-integration-y.e.f
+      .
+      .
+      .
+    provisioning:
+      pip:
+        .
+        .
+        .
+  namespace.tendrl.integration.monitoring:
+    version_bindings:
+      node_agent-x.a.b : node_monitoring-y.e.f
+      .
+      .
+      .
+    provisioning:
+      pip:
+        .
+        .
+        .
+  include /tendrl_integrations.d/*
+
+* Similarly
+  ** ceph-integration will load its definitions in
+     /tendrl_integrations.d/ceph_integration/
+  ** gluster-integration will load its definitions in
+     /tendrl_integrations.d/gluster_integration/
+  ** performance_monitoring will load its definitions in
+     /tendrl_integrations.d/monitoring_integration/
+
+* As part of execution of job in etcd, the flow execution framework in
+  node_agent reads the master definitions in /tendrl_definitions in etcd
+  and subsequently reads all definitions in /tendrl_integrations.d/ on
+  seeing the line "include /tendrl_integrations.d/*".
+
+
+=== Alternatives
+
+* Append monitoring related definitions yaml to node_agents definitions
+  directory in etcd.
+
```

Comment:
Please fix the formatting here. Don't think blank lines are allowed between list items.